artifacts-download-trigger/bin/downloader/action.yml

```diff
@@ -0,0 +1,57 @@
+name: artifact downloader
+description: downloader and extract an artifact associated with given workflow and commit
+author: Josef Fröhle <github@josef-froehle.de>
+branding:
+  icon: download
+  color: green
+inputs:
```

Comment:
(not sure, if this is the right spot for a feature request)
Would it be possible to have an input like `allow-not-found`, because in https://github.com/greenbone/notus-generator/blob/6934f3ea698c05744a2596f69adf4a32398b90f3/.github/workflows/generate.yml#L59 we don't always have an artifact available (which is fine/expected) and would like the pipeline to not raise any warnings in that case.
If this input is `true`, the step should try to download an artifact and always return with `0`, even if it hasn't been found.